composer/utils/eval_client/eval_client.py

```diff
@@ -19,12 +19,10 @@ def invoke(self, payload: Dict[str, str]) -> bool:
         """Invoke a provided dictionary payload to the client.
 
         For code generation, the payload has format:
-        {
             'code': <code to be evaluated>,
             'input': <test input>,
             'output': <test output>,
             'entry_point': <entry point>,
-        }
```

Comment:
Should you leave {}?

Reply:
docs build was failing b/c of like indentation constraints

Reply:
don't understand why, but this fixed it

Reply:
Hm... it looks a bit strange though bc its not clear its a json?

Reply:
fixed